test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
This discussion may be better suited for a new issue, but could we mock some kind of general response (at least for non-grpc calls) via `mitm` that would allow us to traverse all the layers?